README.md

```diff
@@ -11,3 +11,44 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$
+
+## Plagarism Statement
+
+All exercises must contain the following statement:
+“I certify that I have listed all sources used to complete this exercise, including the use
+of any Large Language Models. All of the work is my own, except where stated
+otherwise. I am aware that plagiarism carries severe penalties and that if plagiarism is
+suspected, charges may be filed against me without prior notice.”
+
+## Note
+I am attempting this from scratch, I did however, check it against last semesters answer before submitting. I am trying to work on conciseness.
+
+## Answer
+Start with a substitution (just have to pick either $\log_{2} n$ or $\log_{5} n$ but it doesn't matter which one), this will look like:
+
+$T(n) \in O(\log_{2}(n)) \iff \exists c, n_0: T(n) \leq c*\log_{2}(n) \forall n \geq n_0 $
+
+Then we use the change of base formula $\log_{a}(b)= \frac{(\log_{c}(b))}{(log_{c}(a))}$ (reminded of this by looking at past repository)
+
+This changes the latter portion to $T(n) \leq c*\frac{(\log_{5}(n))}{(log_{5}(2))}$
+
+Which is the same as $T(n) \leq c*\frac{1}{log_{5}(2)} * \log_{5}(n)$
+
+And we can alter it to combine the two constant portions $T(n) \leq \frac{c}{log_{5}(2)} * \log_{5}(n)$
+
+ Which simplifies down to $T(n) \leq c*\log_{5}(n)$ 
```

Comment:
Hang on, I don't think that you can simplify this way -- the $c$ here is clearly not the same $c$ as above.

Reply:
I apologize for my confusion, but do you think I was more correct in the previous commit than now? I guess I am just looking for a direction because I can't see it myself here.